spec/models/profile_spec.rb

```diff
@@ -323,59 +323,95 @@
     it 'activates a profile after password reset' do
       profile = create(
         :profile,
+        :password_reset,
         user: user,
-        active: false,
-        deactivation_reason: :password_reset,
       )
 
+      expect(profile.activated_at).to eq nil # to change
+      expect(profile.active).to eq false # to change
+      expect(profile.deactivation_reason).to eq 'password_reset' # to change
+      expect(profile.fraud_review_pending?).to eq(false)
+      expect(profile.gpo_verification_pending_at.present?).to eq false
+      expect(profile.initiating_service_provider).to eq nil
+      expect(profile.verified_at).to eq nil
+
       profile.activate_after_password_reset
 
-      expect(profile.active).to eq true
-      expect(profile.deactivation_reason).to eq nil
+      expect(profile.activated_at).to be_present # changed
+      expect(profile.active).to eq true # changed
+      expect(profile.deactivation_reason).to eq nil # changed
+      expect(profile.fraud_review_pending?).to eq(false)
+      expect(profile.gpo_verification_pending_at.present?).to eq false
+      expect(profile.initiating_service_provider).to eq nil
       expect(profile.verified_at).to eq nil
     end
 
     it 'activates a previously verified profile after password reset' do
-      verified_at = Time.zone.now - 1.year
       profile = create(
         :profile,
+        :verified,
+        :password_reset,
```

Comment:
I like that you added more standard factory fields here, so that if we need to change initial conditions later, we can do it in one place.

Reply:
yes, will continue this practice, but use caution so as not to preserve corner cases when fields are set ad hoc intentionally. cc/@jmhooper